interface-elements-for-web/articles/spreadsheet/columns-and-rows/specify-column-width-and-row-height.md

```diff
@@ -5,19 +5,29 @@ legacyId: 18150
 ---
 # Specify Column Width and Row Height
 ## Set the column width
-In the **Spreadsheet**, the column width can vary from zero to 255 characters of the default font specified by the predefined _Normal_ style. If you set the column width to zero, the column will be hidden.
+In the **Spreadsheet**, the column width should be between zero and 1785 points. If you set the column width to zero, the column is hidden.
 
-To change the column width, do one of the following.
-1. Drag the right boundary of the column heading to resize the column width. To set the width of multiple columns, select these columns, and then drag the right boundary of any selected column.
-2. To change the column width to automatically fit the contents, select the column and click the **Format** button on the **Home** tab in the **Cells** group. Then, select the **AutoFit Column Width** item from the drop-down list.
+To change the column width, do one of the following:
```

Comment:
This file reads 'To do this', 'To do this', 'To do this'. I suggest changing this to prevent this repetition.